airbyte-integrations/connectors/source-sage-hr/metadata.yaml

```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "https:"
```

Comment:
Reminder to fix this later.

Reply:
sorry, I am not understand, what you are asking to change